package exception.trycatch;
import java.util.Scanner;

public class TryCatchExample2 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		try {
			System.out.println("성함을 적어주세요 :");
			String name = scanner.nextLine();
			System.out.println("성함 :" + name);
		}
		catch(Exception e){
			System.out.println("잘 못 기입하셨습니다. 다시 기입해 주세요");
			System.out.println("성함을 기입해 주세요 :");
			String name2 = scanner.nextLine();
			System.out.println("성함 :"+name2);
		}
		finally {
			System.out.println("환영합니다.");
		}
		System.out.println("종료");
		

	}

}
